siliconcompiler/report/streamlit_viewer.py

```diff
@@ -507,12 +509,33 @@ def show_flowgraph(chip):
     Args:
         chip (Chip) : The chip object that contains the schema read from.
     '''
-    nodes, edges = get_nodes_and_edges(chip, report.get_flowgraph_edges(chip),
-                                       report.get_flowgraph_path(chip))
-    config = Config(width='100%', directed=True, physics=False, hierarchical=True,
-                    clickToUse=True, nodeSpacing=150, levelSeparation=100,
-                    sortMethod='directed')
-    node_from_flowgraph = agraph(nodes=nodes, edges=edges, config=config)
+
+    if os.path.isfile(f'{chip._getworkdir()}/{chip.design}.png'):
+        flow_tab, design_tab, view_file_tab = streamlit.tabs(["Flowgraph", "Design Preview",
+                                                              "View File"])
```

Comment:
I think the file should go next to the manifest tab (the files can be very log and the thin area on the side is too narrow to make it useful).